counterpoll/main.py

```diff
@@ -123,6 +124,55 @@ def disable():
     port_info['FLEX_COUNTER_STATUS'] = DISABLE
     configdb.mod_entry("FLEX_COUNTER_TABLE", PORT_BUFFER_DROP, port_info)
 
+# Ingress PG drop packet stat
+@cli.group()
+@click.pass_context
+def pg_drop(ctx):
```

Comment:
The context can be created in top-level command, please see the context usage and how it helps in UT in show command:
https://github.com/Azure/sonic-utilities/blob/master/show/main.py#L150